uploader.v2.json

```diff
@@ -127,6 +132,11 @@
             "pattern": [],
             "symbolic_name": "dmidecode"
         },
+        {
+            "command": "/usr/sbin/dmidecode -s system-uuid",
+            "pattern": [],
+            "symbolic_name": "bios_uuid"
+        },
```

Comment:
@jhjaggars you have this entry in here twice, did you mean for it to be `etc_machine_id`?